blueprints/ember-inflector/index.js

```diff
@@ -8,6 +8,6 @@ module.exports = {
   },
 
   afterInstall: function() {
-    return this.addBowerPackageToProject('ember-inflector', '1.3.0');
+    return this.addBowerPackageToProject('ember-inflector', '1.4.0');
```

Comment:
no, we shouldn't force this as a bower-install, people should just consume the es6 directly.